Gemfile

```diff
@@ -21,19 +21,30 @@ gem 'turbolinks'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
-gem 'sdoc', '~> 0.4.0',          group: :doc
+gem 'sdoc', '~> 0.4.0', group: :doc
 
-# Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-gem 'spring',        group: :development
-
-#for user authentication
+# design
+gem "font-awesome-rails"
+gem 'bourbon'
```

Comment:
If we don't want `bourbon` to update, we should specify a specific version (as in `sdoc` above.